src/export_js_proxy.cpp

```diff
@@ -285,8 +283,13 @@ namespace pyjs
             {
                 return py::buffer_info(self.m_data, self.m_bytes_per_element, self.m_format_descriptor,
                                         1, {self.m_size}, {self.m_bytes_per_element});
+            })
+            .def("tobytes", [](TypedArrayBuffer& self) {
+                return py::reinterpret_steal<py::object>(
+                    PYBIND11_BYTES_FROM_STRING_AND_SIZE(reinterpret_cast<const char*>(self.m_data), self.m_size)
```

Comment:
I wonder if it would have been enough to do
```C++
.def("tobytes", [](TypedArrayBuffer& self) {
    return py::cast(self)
}
```
since it is already following the buffer protocol

Reply:
This is what I get when doing so: `TypeError: object of type 'pyjs.TypedArrayBuffer' has no len()`

Reply:
ok, then lets just merge it as it is!